src/utils/status/host/constants.js

```diff
@@ -1,5 +1,74 @@
 export const HOST_STATUS_READY = 'HOST_STATUS_READY';
+export const HOST_STATUS_DISCOVERED = 'HOST_STATUS_DISCOVERED';
```

Comment:
I was thinking more about defining the states as constants this way:
`export const HOST_STATUS_VALIDATION_ERROR = 'validation error'`...

And then
```
export const statusTextMap = {
 [HOST_STATUS_VALIDATION_FAILED]: 'Validation Error(s)',
 ...
}
```

And getStatusText doing:
```
return statusTextMap[hostStatus] || hostStatus
```

Reply:
Done